deployment/buildAndDeploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [[ $TRAVIS_BRANCH == 'master' ]]; then
+  ./gradlew clean build -x :keanu-python:build -i uploadArchives -PnexusUser=$NEXUS_USER -PnexusPassword=$NEXUS_PASSWORD -Psigning.keyId=$SIGNING_KEY_ID -Psigning.password=$SIGNING_PASSWORD -Psigning.secretKeyRingFile=../deployment/secret-keys-keanu.gpg --info --stacktrace
```

Comment:
`-i` is a duplicate of `--info`. Also I'd prefer to see all the tasks together, i.e. `uploadArchives` should come directly after `build`

Trying out the funky new "suggestion" feature on github:

```suggestion
  ./gradlew clean build -x :keanu-python:build uploadArchives -PnexusUser=$NEXUS_USER -PnexusPassword=$NEXUS_PASSWORD -Psigning.keyId=$SIGNING_KEY_ID -Psigning.password=$SIGNING_PASSWORD -Psigning.secretKeyRingFile=../deployment/secret-keys-keanu.gpg --info --stacktrace
```